care/utils/enum_choices.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+
+
+class EnumChoices:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too few public methods (1/2)](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5316739)

Reply:
Currently, no more methods are required. They will be added as and when the requirement arises.